src/components/10-atoms/checkbox/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@axa-ch/checkbox",
+  "version": "1.0.0-alpha.0",
+  "description": "The AXA Checkbox component",
+  "author": "Pattern Warriors",
+  "homepage": "https://github.com/axa-ch/patterns-library/tree/master/src/components/10-atoms/checkbox#readme",
```

Comment:
here please use `../tree/develop/..` instead of `../tree/master/..`. This is also fixed in the todays PR of scaffolding

Reply:
fixed